tests/globalhelper/helper.go

```diff
@@ -361,3 +363,19 @@ func CopyFiles(src string, dst string) error {
 
 	return nil
 }
+
+var seededRand *rand.Rand = rand.New(
+	rand.NewSource(time.Now().UnixNano()))
+
+func GenerateRandomString(length int) string {
+	charset := "abcdefghijklmnopqrstuvwxyz"
+
+	//nolint:varnamelen
+	b := make([]byte, length)
```

Comment:
Why not add the following functions to a utility package:
GenerateRandomString
OverrideReportDir
OverrideTnfConfigDir
function in BeforeEach and in AfterEach

Reply:
never mind , the duplication is due to the multiple PRs